Util.js

```diff
@@ -1,6 +1,9 @@
 /* eslint no-unused-vars: ["error", { "varsIgnorePattern": "_" }] */
 /* globals Utilities */
 
+var regexPath_ = /^projects\/.+?\/databases\/\(default\)\/documents\/.+\/.+$/
```

Comment:
Could you add comments for both of these patterns, explaining how they are used in the library?